databricks/koalas/frame.py

```diff
@@ -9634,7 +9674,7 @@ def info(self, verbose=None, buf=None, max_cols=None, null_counts=None):
                 self.count = count_func
 
     # TODO: fix parameter 'axis' and 'numeric_only' to work same as pandas'
-    def quantile(self, q=0.5, axis=0, numeric_only=True, accuracy=10000):
+    def quantile(self, q=0.5, axis=0, numeric_only=True, accuracy=10000) -> Frame:
```

Comment:
Shall we use `Union["DataFrame", "ks.Series"]`?

Reply:
Sounds good, modified!